.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig <https://EditorConfig.org>
+root = true
+
+# elementary defaults
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = tab
+indent_style = space
+insert_final_newline = true
+max_line_length = 80
+tab_width = 4
+​
+# Markup files
+[{*.html,*.xml,*.xml.in,*.yml}]
```

Comment:
Shall we add JSON here too?

Reply:
As a 2 space tab width?

Reply:
Yeah

Reply:
I'll go ahead and make the change, then merge this.

Reply:
```suggestion
[{*.html,*.xml,*.xml.in,*.yml,*.json}]
```